columnphysics/icepack_therm_itd.F90

```diff
@@ -1852,15 +1853,15 @@ subroutine add_new_ice (ncat,      nilyr,      &
                enddo
             endif
 
-           frazil_conc = c0
```

Comment:
This line is still present in a similar section of code, above.  The code is hard-wired for n_iso=3, apparently, so maybe this line should stay in case for some reason n_iso > 3?